Game/Src/Objects/sword.tscn

```diff
@@ -1,47 +1,14 @@
-[gd_scene load_steps=7 format=2]
+[gd_scene load_steps=3 format=2]
 
 [ext_resource path="res://Src/Objects/sword.gd" type="Script" id=1]
 [ext_resource path="res://Assets/Objects/Sword.png" type="Texture" id=2]
-[ext_resource path="res://Assets/Objects/Sword-EpicMax.png" type="Texture" id=3]
-[ext_resource path="res://Assets/Objects/Sword-Rust2.png" type="Texture" id=4]
-[ext_resource path="res://Assets/Objects/Sword-Epic1.png" type="Texture" id=5]
-[ext_resource path="res://Assets/Objects/Sword-Rust1.png" type="Texture" id=6]
 
 [node name="sword" type="KinematicBody2D"]
 script = ExtResource( 1 )
 
 [node name="swordsprite" type="Sprite" parent="."]
-position = Vector2( 198, -192 )
+position = Vector2( 82, -4 )
```

Comment:
For readability, I would define sword_start_x and sword_start_y or something like that on top and use them here

Reply:
This is into godot's scene language. I think it does not matter as we do not see it in the editor.